pkg/endpoint/endpoint.go

```diff
@@ -2077,7 +2077,7 @@ func (e *Endpoint) UpdateLabelsFrom(oldLbls, newLbls map[string]string, source s
 	oldLabels := labels.Map2Labels(oldLbls, source)
 	oldIdtyLabels, _ := labelsfilter.Filter(oldLabels)
 
-	err := e.ModifyIdentityLabels(source, newIdtyLabels, oldIdtyLabels)
+	err := e.ModifyIdentityLabels(source, newIdtyLabels, oldIdtyLabels, 0)
```

Comment:
This case traces up to `updateK8sPodV1` which could plausibly have a similar problem to the namespaces problem you mentioned. I think that for this case it would also make sense to configure `option.Config.CiliumIdentityMaxJitter`

Reply:
ok, I will modify it in next rebase.